### ISSUE TITLE
shall we begin the match later in 6 blocks

### ISSUE THREAD

**Issue by l1t1** [gcp/leela-zero#716]
I saw a lot of match whose rate are very low .

**Comment by Splee99** [comment]
I'm not a neural network expert, but I know that the current network was trained from about one million human games. Those weights may take another million self-play games to get corrected. Or may be we have to increase learning rate? Just my opinion.

**Comment by killerducky** [comment]
There are no human games in the current network. It was trained from the games played by the 5x64 network.

**Comment by Splee99** [comment]
OK, got it. I was mislead by the marker 2.86M+1.52M for the weight ed002c53.

**Comment by Mardak** [comment]
What part of "2.86M+1.52M" made you think it was trained from human games?
